Observer/Customer/Save.php

```diff
@@ -63,11 +63,17 @@ public function execute(Observer $observer)
         if ($customerAddress) {
             $data = array_merge($data, [
                 'country' => $customerAddress->getCountryId(),
-                'state' => $customerAddress->getRegion()->getRegionCode(),
+                'state' => '',
```

Comment:
Would it be better to leave this out or use `null`?

Reply:
@codymullins Good catch.  We set $data['state'] to null initially, so there's no reason to redeclare it here.